### DISCUSSION TITLE
Just runs the base model instead of the trained one? (0 textual inversion embeddings)

### DISCUSSION THREAD

**Discussion by alexfurm7** [TheLastBen/fast-stable-diffusion#956] [Q&A]
Thanks for creating such a powerful tool! I ran it once and got decent results on SD 2.0 (20 pics, 2000 UNet, 350 text_encoder). Been trying to rerun it several times to get better results (adding steps to Unet, switching to v1.5), but every time it seems like the model is just the base model. When I go to test it, the Stable Diffusion outputs are not at all related to my sample images. I've tried clearing out the SD folder from my google drive in case there was some old file. Also tried reseting the session in Colab (and upgrading Colab).

I noticed the following output line when I run the step to Test the Trained model:

```
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading weights [7140137c] from /content/gdrive/MyDrive/Fast-Dreambooth/Sessions/[[my session folder]]/[[my ckpt file]]
Applying xformers cross attention optimization.
Model loaded.
Loaded a total of 0 textual inversion embeddings.
Embeddings: 
Running on local URL:
```

No errors otherwise. I see the cpkt file created in my google drive.

**Comment by TheLastBen** [comment]
why switch to v1.5 when training the v2?

**Reply by alexfurm7** [reply]
Started on v2 at 3000 steps. Wanted to improve it so tried "resume training", which didn't work and was the first time I encountered the model not being trained at all. Then, after reading mixed reviews about v2 while waiting for models to load, I retried the entire process from to 1.5 for model download, training, and testing. No matter how I run it, I can't seem to get it to stop using the default model.

**Reply by TheLastBen** [reply]
maybe you forget to check the box "resume training", it should write in green "resuming training" in the output

**Reply by alexfurm7** [reply]
I've tried that, too. But since then, I've basically just scrapped everything and started from scratch. New session. New SD folder. New training. But still just getting base model. Are the "0 textual inversion embeddings" indicative of an error? Wondering if maybe the training was not able to get anything from my training set of 20 images?

**Reply by TheLastBen** [reply]
0 textual inversion embeddings is not related to dreambooth, it's a normal message, make sure you rename the instance images to one unique non-existing word and use that word in A1111 to summon the trained sbject

**Reply by alexfurm7** [reply]
Ok thank you @TheLastBen . Really appreciate you building and maintaining this. I'm going to try resetting everything. New images, back to v2. See what happens :)

**Reply by alexfurm7** [reply]
@TheLastBen ok, this is weird. I started from scratch. All worked fine. The first couple of renderings were clearly done by the fine tuned model based on the images provided. But with each time I ran the renderings, the results became more generalized. Less fine-tuned. They would deviate more and more from the training set. Any ideas?

**Reply by TheLastBen** [reply]
Try different samplers and keep the cfg scale below 9, the model can't change without getting trained again.
